include/seqan3/range/decorator/gap_decorator_anchor_set.hpp

```diff
@@ -67,10 +66,16 @@ namespace seqan3
  * Resolving random access requires logarithmic access into the set and inserting or removing a gap symbol additionally
  * entails updating all subsequent elements in the set to preserve correct cumulative sizes.
  *
+ * ### The seqan3::gap_decorator_anchor_set::iterator type
+ *
+ * \attention The iterator of the seqan3::gap_decorator_anchor_set does not model the std::LegacyInputIterator because
```

Comment:
```suggestion
 * \attention The iterator of the seqan3::gap_decorator_anchor_set does not model the [Cpp17InputIterator](https://en.cppreference.com/w/cpp/named_req/InputIterator) because
```

Reply:
I had to learn that there is no std::LegacyInputIterator entry in cppreference. So it needs to be linked explicitly and by doing it we can also give it the name as used in the standard.

Reply:
Also I think you still should mention that the return type is not  a reference but a proxy. I think it is more important than a missing operator->.